spec/support/record_shared.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
-describe GapIntelligence::Record do
-  it 'allows records to be created' do
+shared_examples_for 'Record' do
```

Comment:
I would update all classes inherited from Record to include this shared example, so that we can keep consistency of the change.